plugins/carbon_black_defense/help.md

```diff
@@ -230,7 +230,7 @@ This action is used to retrieve details for an individual event given the event
 
 |Name|Type|Default|Required|Description|Enum|Example|
 |----|----|-------|--------|-----------|----|-------|
-|event_ids|string|None|True|Event ID|None|422af3fc3a7411ea8da649e797467dc0|
+|event_ids|string|None|True|Event ID used to retrieve event details. Example: 422af3fc3a7411ea8da649e797467dc0|None|422af3fc3a7411ea8da649e797467dc0|
```

Comment:
Make sure example is only in example field and not in description field

Reply:
Updated to include in description field as part of ticket request.